<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_p2p</class>
 <widget class="QWidget" name="Widget_p2p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="9">
    <widget class="QTextBrowser" name="messageBrowser"/>
   </item>
   <item row="2" column="0">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="sizeComboBox">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="boldToolBtn">
     <property name="toolTip">
      <string>加粗</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/bold.png</normaloff>:/new/icon/images/bold.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="italicToolBtn">
     <property name="toolTip">
      <string>倾斜</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/italic.png</normaloff>:/new/icon/images/italic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="underlineToolBtn">
     <property name="toolTip">
      <string>下划线</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/under.png</normaloff>:/new/icon/images/under.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="colorToolBtn">
     <property name="toolTip">
      <string>更改字体颜色</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/color.png</normaloff>:/new/icon/images/color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="3">
    <widget class="QToolButton" name="sendToolBtn">
     <property name="toolTip">
      <string>传输文件</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/send.png</normaloff>:/new/icon/images/send.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QToolButton" name="saveToolBtn">
     <property name="toolTip">
      <string>保存聊天记录</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="IconSources.qrc">
       <normaloff>:/new/icon/images/save.png</normaloff>:/new/icon/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QToolButton" name="clearToolBtn">
     <property name="toolTip">
      <string>清空聊天记录</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../chat/images/clear.png</normaloff>../chat/images/clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="11">
    <widget class="QTextEdit" name="messageTextEdit">
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="4">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="id">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>对方ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLabel" name="nickname">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>对方昵称:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IconSources.qrc"/>
 </resources>
 <connections/>
</ui>
